### PR TITLE
DO-5737 node version upgrade errors

### PR DESCRIPTION
**Pull request by ojintoad**
https://ngpvan.atlassian.net/browse/DO-5737

Curl by default doesn't follow redirects. When it doesn't, it downloads a 0 byte file. This adds the -L argument so it will.

Search `-L, --location` for more:
https://www.man7.org/linux/man-pages/man1/curl.1.html

and the Jira item has details on how I diagnosed and what articles led me to this conclusion.

@FSQuaratiello tested against dev24 with his upgrade of node this was blocking

I've tested against develop fastaction with develop fastaction code that isn't upgraded to make sure it didn't break that deploy. Unsurprisingly, it did not.

https://build.ngpvan.com/buildConfiguration/AcceleratorDigitalServices_Fastaction_Deploy/3623901?buildTab=log&focusLine=3&linesState=500

image

https://dashboard.heroku.com/apps/ngpvan-fastaction-develop/settings

* [ ] Post merge I need to set develop back to the main branch for the buildpack

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -44,7 +44,7 @@ fi
 # Download node from Heroku's S3 mirror of nodejs.org/dist
 status "Downloading and installing node"
 node_url="http://s3pository.heroku.com/node/v$node_version/node-v$node_version-linux-x64.tar.gz"
```
Comment (starwed) [inline bin/compile]: Very much a drive-by comment, but can this use https instead?

**Review by starwed — APPROVED** [review]

**Review by ghost — APPROVED** [review]

File: bin/compile (hunk 1 above)
Comment (ojintoad) [reply bin/compile]: ha ha this is totally reasonable - let me test later today and if it seems to work with existing and new version i'll modify. Good catch.

Comment (ojintoad) [reply bin/compile]: worked fine locally, will run the same test as yesterday for fa develop and merge

**Comment by ojintoad** [comment]
https update worked too https://build.ngpvan.com/buildConfiguration/AcceleratorDigitalServices_Fastaction_Deploy/3626239?buildTab=log&focusLine=18

merging @FSQuaratiello you shouldn't need to specify a buildpack any longer after this merge
